Run clang-format for torch/distributed/rpc

pietern opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#27531 Run clang-format for torch/distributed/rpc**
* #27530 Rename PythonUDF{Call,Resp}
* #27529 Scope pybind11 functions to torch.distributed.{autograd,rpc}

Differential Revision: [D17808206](https://our.internmc.facebook.com/intern/diff/D17808206)

pietern committed:
Run clang-format for torch/distributed/rpc

pietern committed:
Update on "Run clang-format for torch/distributed/rpc"

mrshenli reviewed (approved):
Stamp the changes on format change, but this PR needs a rebase to update message types.

pietern committed:
Update on "Run clang-format for torch/distributed/rpc"


Differential Revision: [D17808206](https://our.internmc.facebook.com/intern/diff/D17808206)

pietern committed:
Update on "Run clang-format for torch/distributed/rpc"


Differential Revision: [D17808206](https://our.internmc.facebook.com/intern/diff/D17808206)

pietern committed:
Update on "Run clang-format for torch/distributed/rpc"


Differential Revision: [D17808206](https://our.internmc.facebook.com/intern/diff/D17808206)

pietern committed:
Update on "Run clang-format for torch/distributed/rpc"


Differential Revision: [D17808206](https://our.internmc.facebook.com/intern/diff/D17808206)

pietern commented:
The commit that's landing has a fix for the build failures in the most recent commit in this PR.
